### PR TITLE
Contact labels

### PR DESCRIPTION
**Pull request by evasilkovich**

### CODE REVIEW

File: client/src/components/contact-update/index.vue [hunk 1]
```diff
@@ -181,6 +187,7 @@ export default {
       if (this.group && this.name) {
         api.getOne(this.group, this.name)
           .then(data => {
+            debugger
```
Comment (eorlovsky) [inline client/src/components/contact-update/index.vue]: remove that

File: client/src/components/contact-details/index.vue [hunk 2]
```diff
@@ -60,6 +60,18 @@
...
+                <md-chips v-model="labels_names" md-static></md-chips>
+                <div class="md-list-item-text">
+                  <span></span>
+                  <span>Labels</span>
+                </div>
+              </div>
+
+            </md-list-item>
+
             <md-divider></md-divider>
```
Comment (eorlovsky) [inline client/src/components/contact-details/index.vue]: Try to remove this <divider> as I looks bad 
screen shot 2018-08-15 at 11 15 27 am

File: server/serverless.yml [hunk 3]
```diff
@@ -216,6 +240,24 @@ functions:
...
+
+  labels-group-list:
+    handler: index.handler
+    events:
+      - http:
+          path: '/groups/{group_id}/labels'
+          method: get
+          cors: true
+
+  init-groups:
```
Comment (eorlovsky) [inline server/serverless.yml]: How this function is invoked ?

Comment (evasilkovich) [reply server/serverless.yml]: router.get('', function (req, res) {
  console.log('groups-list - starting')
 
and then 

  getGroups()
    .then(ifGroupsExist)        // return response
    .then(invokeInitGroups)  // invoke init-group function
    .then(getGroups)

Comment (eorlovsky) [reply server/serverless.yml]: No this is not correct. AWS CloudFormation can run AWS Lambda function on deploy.

**Pull request merged**